azurerm/service.go

```diff
@@ -17,3 +17,12 @@ var (
 		VirtualMachines.String(): struct{}{},
 	}
 )
+
+// GetSupportedServices returns all the AWS service names that Terracost supports.
+func GetSupportedServices() []string {
+	svcs := make([]string, 0, len(services))
+	for k := range services {
+		svcs = append(svcs, k)
+	}
+	return svcs
+}
```

Comment:
I would `return slices.Sort(svcs)` as this is iterating a Map so the order is random, even if now there is only 1 value